lib/components/base/avatar/MAvatar.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * @description vue version avatar
+ * @author Muzych
+ * @date 2024/03/03 00:40
+ * @version v1.0.0
+ *
+ * 江湖的业务千篇一律，复杂的代码好几百行。
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import { AvatarProps } from './index';
+import { MC, Slot } from '../../../types';
+import './avatar.css';
+
+export default function MAvatar(props: AvatarProps & Slot & MC) {
```

Comment:
`MC` used for penetrating class like this:
``` ts
className: clsx([props.className, 'm-button', { 'm-button-disabled': props.disabled }, `m-button-${props.type ?? 'default'}`]),
```
The key point is `props.className`